pkg/cleanup/cleanup.go

```diff
@@ -62,6 +65,40 @@ func GetInactiveTags(activeTags, tags *[]string) []string {
 	return inactiveTags
 }
 
+// GetOrphanTags returns the tags that does not have any git commit match
+func GetOrphanTags(values, tags *[]string, matchOption MatchOption) []string {
```

Comment:
This project is very unfortunate, as the word "tag" is used both in "git tag" as well as "image tag". Please name it clearly `GetOrphanImageTags` to avoid confusion, as recommended by Clean Code. Also, what is `values`?

Reply:
The values are either git tags or git commits